examples/custom_configuration_auto_register/system/container.rb

```diff
@@ -1,6 +1,10 @@
 require 'dry/system/container'
 
 class App < Dry::System::Container
+  configure do |config|
+    config.auto_registration.memoize = true
```

Comment:
We should probably update this example too 😄

Reply:
Oops! I totally forgot about this!